TypeCobol/Tools/Options-Config/TypeCobolConfiguration.cs

```diff
@@ -149,7 +150,54 @@ public enum OutputFormat {
         Cobol85Nested,
         Documentation
     }
-    public static class TypeCobolOptionSet
+
+    public class TypeCobolCheckOption
```

Comment:
Several remarks on this class :
- do not use tabs for indent
- constructor taking a boolean is ambiguous because the resulting Severity is not known by caller

I think we should only have a single constructor taking a Nullable<Severity> as input and a public static Parse method to create a TypeCobolCheckOption from a string.